omaha/goopdate/download_manager_unittest.cc

```diff
@@ -1177,6 +1192,53 @@ TEST_F(DownloadManagerUserTest, GetPackage) {
   EXPECT_SUCCEEDED(DeleteDirectory(dir));
 }
 
+TEST_F(DownloadManagerUserTest, CachePackage) {
+  ASSERT_SUCCEEDED(RegKey::SetValue(MACHINE_REG_UPDATE_DEV,
+                                    kRegValueAlwaysVerifyAuthenticodeSignatures,
+                                    (DWORD)1));
+
+  const TCHAR* kFiles[] = {_T("SaveArguments.exe"),
+                           _T("old_google_certificate.dll"),
+                           _T("sha2_0c15be4a15bb0903c901b1d6c265302f.msi")};
+  HRESULT kExpected[] =   {S_OK,
```

Comment:
There is some extra white spacing here.

Reply:
Fixed in 497b5a7e0abcccea3c2bacdf3ad69df2c46c8c42.